doc/nrf/releases/release-notes-2.4.0.rst

```diff
@@ -0,0 +1,895 @@
+
+* :ref:`peripheral_hids_mouse` sample:
+
+  * Updated the sample to include the :kconfig:option:`CONFIG_BT_SMP` Kconfig option when ``CONFIG_BT_HIDS_SECURITY_ENABLED`` is selected.
+  * Fixed a CMake warning by moving the nRF RPC configuration (the :kconfig:option:`CONFIG_NRF_RPC_THREAD_STACK_SIZE` Kconfig option) to a separate overlay config file.
+
+* :ref:`direct_test_mode` sample:
+
+  * Added:
+
+    * Support for the :ref:`nrfxlib:mpsl_fem` Tx power split feature.
+      The DTM command ``0x09`` for setting the transmitter power level takes into account the front-end module gain when this sample is built with support for front-end modules.
+      The vendor-specific commands for setting the SoC output power and the front-end module gain are not available when the :kconfig:option:`CONFIG_DTM_POWER_CONTROL_AUTOMATIC` Kconfig option is enabled.
+    * Support for +1 dBm, +2 dBm, and +3 dBm output power on the nRF5340 DK.
+
+  * Updated the handling of the hardware erratas.
+  * Removed a compilation warning when used with minimal pinout Skyworks FEM.
+
+* :ref:`peripheral_uart` sample:
+
+  * Fixed the unit of the :kconfig:option:`CONFIG_BT_NUS_UART_RX_WAIT_TIME` Kconfig option to comply with the UART API.
+
+* :ref:`nrf_dm` sample:
+
+  * Updated the sample by improving the scalability of the sample when it is being used with more devices.
+
+* :ref:`peripheral_fast_pair` sample:
+
+  * Added the default Fast Pair provisioning data that is used when no other provisioning data is specified.
+  * Updated the documentation to align it with the new way of displaying notifications for the Fast Pair debug Model IDs.
+
+* Removed the Bluetooth: External radio coexistence using 3-wire interface sample because of the removal of the 3-wire implementation.
+
+Bluetooth mesh samples
+----------------------
+
+* Added:
+
+  * Samples :ref:`ble_mesh_dfu_target` and :ref:`ble_mesh_dfu_distributor` that can be used for evaluation of the Bluetooth mesh DFU specification and subsystem.
+  * :ref:`bluetooth_mesh_light_dim` sample that demonstrates how to set up a light dimmer and scene selector application.
+* Updated the configuration of advertising sets in all samples to match the new default values.
+  See `Bluetooth mesh`_ for more information.
+* Removed the :file:`hci_rpmsg.conf` file from all samples that support nRF5340 DK or Thingy:53.
+  This configuration is moved to the :file:`ncs/nrf/subsys/bluetooth/mesh/hci_rpmsg_child_image_overlay.conf` file.
+* :ref:`bluetooth_mesh_light_lc` sample:
+
+  * Updated to demonstrate the use of sensor server to report additional useful information about the device.
+
+* :ref:`bluetooth_mesh_sensor_server` and :ref:`bluetooth_mesh_sensor_client` samples:
+
+  * Added:
+
+    * Support for motion threshold as a setting for the presence detection.
+    * Support for ambient light level sensor.
+    * Shell support to :ref:`bluetooth_mesh_sensor_client` sample.
+
```

Comment:
```suggestion

Cryptography samples
--------------------

* Updated all samples and their documentation as the supported logging backend changed to UART instead of both SEGGER RTT and UART.

```